robot_skills/src/robot_skills/arm/arms.py

```diff
@@ -155,6 +155,12 @@ def handover_detector(self):
         self._test_die(hasattr(self._arm, 'handover_detector'), "This arm does not have a handover_detector")
         return self._arm.handover_detector
 
+    @property
+    def gripper_position_detector(self):
```

Comment:
What does a `gripper_position_detector` do? We already know where the gripper is with the joint_states/TF.

Reply:
So how open the fingers are?

Reply:
Yes

Reply:
Yes. And upon further inspection we can use the get_joint_states function of the robot object for this but this allows us to already specify which joint corresponds to the fingers. That is something that can potentially be improved. But for now we want to see if this gives us the desired behaviour. Lets not do premature optimization.

Reply:
But this sounds like a property of the gripper, not the arm. Which also raises the question whether the handover detector should be part of the arm or the gripper.

Reply:
That is a different can of worms and not for this project.

Reply:
The `handover_detector` can be ignored in this project, but `gripper_position_detector` should be modelled correctly. It doesn't mean it should be done correctly right now in this PR. But it should be done correctly soon.

Reply:
#stopdeverloederingvanonzecode

Reply:
We should create an issue about this. With a deadline this should be fixed.

Reply:
I created an issue to look into the grasp position detector robot part. It does not have high priority though. So I don't think we need to add a deadline